packages/vm/src/cache.rs

```diff
@@ -88,7 +88,8 @@ where
     pub fn save_wasm(&mut self, wasm: &[u8]) -> VmResult<Checksum> {
         check_wasm(wasm, &self.supported_features)?;
         let checksum = save_wasm_to_disk(&self.wasm_path, wasm)?;
-        let module = compile(wasm)?;
+        const MEMORY_LIMIT: Size = Size::mebi(16);
```

Comment:
Very cool that we can set a memory limit.
I assume this writes it into the artifact which is then read with `load_wasm`.

But where does this constant 16MB come from? (Seems like it is same as the `TESTING_MEMORY_LIMIT`)

Reply:
Well spotted. In this case we don't need a limit at all since the module is never instantialted. 6551573f921eb3a9260a34a6dc49cba86e9e7154 makes this explicit by replacing the fake limit with an `Option`.